sql/pgwire_test.go

```diff
@@ -292,6 +292,9 @@ func TestPGPreparedQuery(t *testing.T) {
 		"SHOW database": {
 			base.Results(""),
 		},
+		"SELECT 1 FROM system.descriptor WHERE descriptor != $1": {
```

Comment:
How about a test where byte arrays are used for the output and not just input?

Is there any way to force the use of text format instead of binary format? (i.e. does the hex encoding currently have any test coverage?)